test/unit/math/prim/prob/discrete_range_ccdf_log_test.cpp

```diff
@@ -0,0 +1,15 @@
+#include <stan/math/prim.hpp>
+#include <gtest/gtest.h>
+
+TEST(ProbDiscreteRange, cdf_log_matches_lccdf) {
```

Comment:
We know the implementation just delegates, but these tests are to prevent future breakage, so they should be as throough as everything else.    One way to do this is something like this:

```
for (int lower = 0; lower < 5; ++lower) {
  for (int upper = lower; upper < 5; ++upper)
    for (int y = lower; y <= upper; ++y)
       ... test that they do the same thing...
```

Same for `discrete_range_cdf`.

Reply:
I've changed tests to do as suggested.